packages/asset-uploader/src/utils/format-asset.ts

```diff
@@ -24,8 +25,19 @@ export const formatAsset = (asset: DbAsset): Asset => {
     };
   }
 
+  let transform: ImageAsset["transform"] = "none";
+
+  if (asset.location === "REMOTE") {
+    transform = "relative";
+
+    if (env.VERCEL_ENV != null) {
```

Comment:
same here, how can we have a flag that is independent of the specific environment name like "vercel", "cloudflare" or whatever